src/hlstrafe.hpp

```diff
@@ -179,23 +178,24 @@ namespace HLStrafe
 	};
 
 	typedef std::function<TraceResult(const float[3], const float[3], HLStrafe::HullType)> TraceFunc;
+	typedef std::function<int(float[3], int*)> PointContentsFunc;
```

Comment:
```suggestion
	typedef std::function<int(const float[3])> PointContentsFunc;
```
We don't use truecont yet and it will be easy to add if we want it.